### ISSUE TITLE
Chrome OS issues

### ISSUE THREAD

**Issue by Test18415** [GoogleChromeLabs/chromeos_smart_card_connector#1234]
This wednesday a new version of the chromeos smart card connector app came out. This update made our kiosk solution crash, 200 devices had downtime. The reason was that chrome os 128+ was needed and we had still an old version of 121+. 

Is it possible to get notified in the future when the smart card connector reader app needs a new version of chrome OS.

**Comment by emaxx-google** [comment]
Hello, apologies for the disruption. Are these existing devices? That would seem like a bug in Chrome, because the documentation says:

> Existing users of your extension won't receive updates when the minimum_chrome_version is higher than their current browser version.

https://developer.chrome.com/docs/extensions/reference/manifest/minimum-chrome-version

**Comment by emaxx-google** [comment]
For our understanding, the Kiosk deployment in question is some other application that has Smart Card Connector as a seconary app?

**Comment by emaxx-google** [comment]
@Test18415: To confirm, is your deployment already mitigated (presumably, by updating ChromeOS)? I'm asking because we considered temporarily lowering `minimum_chrome_version`, but it might be too late to make any real difference.

**Comment by Test18415** [comment]
Yes, we have updated the devices allready, so a change is not needed for us.

**Comment by emaxx-google** [comment]
That's good to know, thanks. FWIW we're still going to lower the `minimum_chrome_version` to 126, to match the current LTS version (the rollout is planned for the next week). In parallel the Chromium team is looking into the Kiosk bug.

**Comment by emaxx-google** [comment]
The kiosk bug is still being investigated. In the meantime we're pushing a 1.7.0 version update now that lowers the `minimum_chrome_version` to 126.

**Comment by bderuijsscher** [comment]
Thanks, we must be able to prevent such downtime for our project. Is it possible to be notified upon changes before the updates to the connector are published? This way, we might be able to test and validate the solution. Or, pinning the connector to a specific version so that we can plan those validations/upgrades ourselves?
